src/gui/mainwindow.cpp

```diff
@@ -1475,6 +1482,33 @@ void MainWindow::showStatusBar(bool show)
     }
 }
 
+void MainWindow::showFiltersSidebar(const bool show)
+{
+    Preferences *const pref = Preferences::instance();
+
+    if (show && !m_transferListFiltersWidget)
+    {
+        const int width = pref->getFiltersSidebarWidth();
+        m_transferListFiltersWidget = new TransferListFiltersWidget(m_splitter, m_transferListWidget, isDownloadTrackerFavicon());
+        m_splitter->insertWidget(0, m_transferListFiltersWidget);
+        m_splitter->setCollapsible(0, true);
+        m_splitter->setSizes({width, (m_splitter->width() - width)});
```

Comment:
`m_splitter->width()` in this method is 1054 for me (_after_ the insertion of `m_transferListFiltersWidget` to the splitter) while in `MainWindow::writeSplitterSettings()` it is 1904 which of course also changes `m_splitter->sizes()` used there. 
No idea where to go from there.

Reply:
Update: can only be reproduced if qBt window is un-maximized and then maximized.